setup.cfg

```diff
@@ -31,11 +31,17 @@ universal = 1
 [tool:pytest]
 minversion = 3.1
 testpaths = tests
-mock_use_standalone_module = true
 filterwarnings =
     ignore::sqlalchemy.exc.SAWarning
 addopts =
     --appini=tests/test.ini
-    --cov=clld
-    --cov-report term-missing
+    --cov
+    --cov-report=term
```

Comment:
Since clld does have coverage close to 100%, I think reporting missing lines makes sense.

Reply:
+1 (see below)